tools/accuracy_checker/accuracy_checker/annotation_converters/__init__.py

```diff
@@ -142,4 +143,5 @@
     'ADE20kConverter',
 
     'LibrispeechConverter',
+    'CriteoKaggleDACConverter',
```

Comment:
@mzuevx , I do not see converter code in the request

Reply:
and it is not my  inattention, because our CI also can not find it:
```
ImportError: No module named 'accuracy_checker.annotation_converters.criteo_kaggle_dac'
```

Reply:
Sorry, it's my mistake. Fixed